src/main/java/seedu/module/logic/parser/ModuleBookParser.java

```diff
@@ -37,7 +38,7 @@
      * @return the command based on the user input
      * @throws ParseException if the user input does not conform the expected format
      */
-    public Command parseCommand(String userInput) throws ParseException {
+    public Command parseCommand(String userInput) throws ParseException, CommandException {
```

Comment:
Is it possible to not throw command exception here? The parser should only be concerned about parser related exceptions, not command exceptions

Reply:
yup sure